extensions/exchanges/bitfinex/exchange.js

```diff
@@ -66,6 +84,48 @@ module.exports = function container (get, set, clear) {
       })
     },
 
+    getTrades2: function (opts, cb) {
+      var func_args = [].slice.call(arguments)
+      var client = publicClient2()
+      var symbol = joinProduct2(opts.product_id)
+      var args = {}
+      args.sort = 1
+      // v2 limit is 1000
+      args.limit = 1000
+      if (opts.from) {
+        // initial value is null
+        args.start = opts.from
+      }
+      if (opts.to) {
+        // started point
+        args.end = opts.to
```

Comment:
in the commands you are setting this to `new Date().getTime()`, why not just do that here?